### DISCUSSION TITLE
Marketing

### DISCUSSION THREAD

**Discussion by ivannaprata** [EasyRoomCompany/Guardian#70] [Q&A]
@gabrielsousarod gostei da ideia de acrescentar um vídeo rápido, estilo vinheta, com a página da EasyRoom. Como está o andamento do plano de marketing?

**Comment by gabrielsousarod** [comment]
já estou com o script do vídeo e hoje começa a gravação da narração e dos takes @kevinDsousa tem algo relacionado ao que vc ia me mandar pra teste para aparecer no vídeo?

**Comment by mikaelmonteirodev** [comment]
Maravilha!

**Comment by kevinDsousa** [comment]
@gabrielsousarod nos criamos algumas telas da aplicação mas ainda estamos inplementando as funcionalidades, vou te enviar algumas telas que o sistema comporta para você dar seguimento na tratativa

![image](https://github.com/HackathonTeam05/Guardian/assets/55944691/9254ee37-6c67-48c9-bfb6-bc391634ba72)

---

![image](https://github.com/HackathonTeam05/Guardian/assets/55944691/490ff020-d33e-4dbd-9427-5afc955b6356)

---

![image](https://github.com/HackathonTeam05/Guardian/assets/55944691/5235ef0a-48f1-4acd-9177-55d3ef376820)

----

![image](https://github.com/HackathonTeam05/Guardian/assets/55944691/f9d4294c-bdca-45d7-a56d-3f4e157253d9)

**Reply by gabrielsousarod** [reply]
obrigado!!!!
